src/components/core/check-overflow/index.js

```diff
@@ -3,9 +3,9 @@ function checkOverflow() {
   const swiper = this;
   const params = swiper.params;
   const wasLocked = swiper.isLocked;
-  const lastSlidePosition = (params.slidesOffsetBefore + (params.spaceBetween * (swiper.slides.length - 1)) + ((swiper.slides[0]).offsetWidth) * swiper.slides.length);
+  const lastSlidePosition = swiper.slides.length > 0 ? (params.slidesOffsetBefore + (params.spaceBetween * (swiper.slides.length - 1)) + ((swiper.slides[0]).offsetWidth) * swiper.slides.length) : undefined;
```

Comment:
Minor detail but if you'd want to shorten this you can do this instead as this would evaluate to `undefined` if length is 0
```
const lastSlidePosition = swiper.slides.length > 0 && (params.slidesOffsetBefore + (params.spaceBetween * (swiper.slides.length - 1)) + ((swiper.slides[0]).offsetWidth) * swiper.slides.length);
```